js/packages/openinference-core/package.json

```diff
@@ -9,10 +9,17 @@
   "description": "OpenInference Core provides utilities shared by all OpenInference SDK packages.",
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "build": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json && tsc-alias -p tsconfig.esm.json",
+    "postbuild": "echo '{\"type\": \"module\"}' > ./dist/esm/package.json; rimraf dist/test",
```

Comment:
no way to ignore the test directory in the build step?

Reply:
It's because we share the same tsconfig for both tests and src, and then build with TSC using that config.

They don't provide an exclude flag at the cli level either.

I filed an issue for this a month or so back, will tackle it in the future

Reply:
See #1077